litmus-portal/graphql-server/utils/misc.go

```diff
@@ -31,55 +31,63 @@ func RandomString(n int) string {
 
 //ManifestParser parses manifests yaml and generates dynamic manifest with specified keys
 func ManifestParser(cluster database.Cluster, template string, subscriberConfig *types.SubscriberConfigurationVars) ([]byte, error) {
-	defaultState := false
+	var (
```

Comment:
I think GitHub have some problem, This file is compared with the old commits.
**No change has been made to this file.**

Reply:
The branch is not updated with master, that could be the reason. Can you update it and check

Reply:
This is not the update problem. Every PR compares with the latest commit in master.